src/sass/layouts/_footer.sass

```
@@ -0,0 +1,95 @@
+@import "./base/variables"
+@import "./base/mixins"
+
+footer
+    .footer__main
+        background-color: $background-grey
+        padding: 60px 10px 0px
+        .footer__top
+            .footer__title
+                @include title
+                display: inline
+                text-align: left
+                margin-bottom: 30px
+            span.icon
+                margin-right: 20px
+                i
+                    font-size: 0.8rem
+                    color: $hover-color
+                    &::before
+                        position: absolute
+                        top: 8px
+            ul.link
+                @include mt-mb(20px, 40px)
+                li.link-item
+                    @include pt-pb(12px, 12px)
+                    a
+                        @include text
+                        font-weight: $regular
+                        i
+                            margin-right: 10px
+
+        .footer__bottom
+            padding-bottom: 60px 
+            border-top: 1px solid $border-color 
+            @include screen(1024px)
+                margin-top: 20px
+            .title
+                @include mt-mb(20px, 20px)
+                @include text
+            .footer__payment
+                img
+                    padding-right: 10px
+            .footer__hotline
```

Viết SCSS/SASS thì đã tách ra riêng từng phần nhỏ khác nhau rồi. Nghĩa là mỗi file chỉ style cho 1 thứ nhất định. Và ngoài ra còn đặt tên class theo tiêu chuẩn BEM nữa nên không cần phải lồng nhau nhiều như này làm gì đâu. Lí tưởng là chỉ nesting tầm 3 hoặc 4 vòng thôi nhé
Nesting lồng nhiều vào nhau khi compile ra file CSS sẽ khiến selector nó bị dài
`.footer__payment {}` là đủ rồi
không cần phải như này
`.footer .footer__bottom .footer__payment {}`
Ngoài ra có thể tìm hiểu thêm công dụng của dấu & trong SCSS/SASS

```suggestion
        .footer__bottom
            padding-bottom: 60px 
            border-top: 1px solid $border-color 
            @include screen(1024px)
                margin-top: 20px
            .title
                @include mt-mb(20px, 20px)
                @include text
        .footer__payment
                img
                    padding-right: 10px
       .footer__hotline
```